src/anomalib/models/efficient_ad/lightning_model.py

```diff
@@ -90,8 +89,8 @@ class EfficientAd(AnomalyModule):
 
     def __init__(
         self,
-        teacher_out_channels: int,
-        image_size: tuple[int, int],
+        input_size: tuple[int, int],
```

Comment:
Same as above, would it be an idea to provide a default value to `input_size`? Maybe we could discuss this?